input/groups/armor.json

```diff
@@ -2167,7 +2170,7 @@
         11404,
         11406
       ],
-      "Reality Rig": [
+      "Reality Rig Mk1 / Reality Rig Mk2 / Reality Rig Mk3": [
```

Comment:
I prefer complete names, specially when they're different tiers.

In this case, it should be three different groups, but kinda pointless with just chest armor, so they get packed together.

Instances of this particular "multiple groups into one" case:

* Adventurer (2 to 1)
* Special Ops / Special Ops Captain (2 to 1)
* Reality Rig Mk1 / Reality Rig Mk2 / Reality Rig Mk3 (3 to 1)
* Aetherized Arsenite / Aetherized Indigo / Aetherized Vermilion (2 to 1)
* Caladbolg (3 to 1)

The idea is to emphasize there's different tiers inside the group, just that.

Still, "Reality Rig" would be fine if you prefer that one, since it's the common name. Same scenario with Special Ops.

Reply:
Fair, I'm not particular fussed.